hooks/pre/10-reset_data.rb

```diff
@@ -176,8 +176,8 @@ def empty_database!(config)
   system(pg_sql_statement(config, delete_statements)) if delete_statements
```

Comment:
We should not use `system()` without checking whether it succeeded

Reply:
Thanks. I've changed this to use `execute()` instead.